### ISSUE TITLE
chip.de

### ISSUE THREAD

**Issue by NanoMeow** [NanoMeow/QuickReports#368]
### Basic Information

**Test link:** `https://www.chip.de/bildergalerie/TeamSpeak-3-perfekt-einrichten-und-nutzen-Galerie_31885346.html`
**Category:** anti-adblock
**Reported from:** Germany
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.102 Safari/537.36
**Extension:** Nano Defender 15.0.0.75

**Comment by jspenguin2017** [comment]
Works fine for me. 
![image](https://user-images.githubusercontent.com/7283682/48982646-86b24b80-f0a2-11e8-983a-24d152f7e265.png)

**Comment by NanoMeow** [comment]
### Basic Information

**Test link:** `https://www.chip.de/downloads/c1_downloads_auswahl_53513907.html?t=1584108268&v=3600&s=2dd207af76eb17dc3da8f58abc6b6751`
**Category:** anti-adblock
**Reported from:** Germany
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 6.2; Win64; x64; rv:74.0) Gecko/20100101 Firefox/74.0
**Extension:** Nano Adblocker 1.0.0.140

### Additional Message

Adblock warning message. It can easily be circumvented (it is possible to choose whether to watch the ad or not), but nevertheless, the adblock warning is annoying and should be removed.

**Comment by jspenguin2017** [comment]
I don't see any issues with this website. 
![image](https://user-images.githubusercontent.com/7283682/76686415-c9be6800-65e0-11ea-90df-dda473f5b6dd.png)

**Comment by NanoMeow** [comment]
### Basic Information

**Test link:** `https://www.chip.de/downloads/c1_downloads_auswahl_53513907.html?t=1584208608&v=3600&s=623c1bb433235ddfd3c25cea81e71328`
**Category:** anti-adblock
**Reported from:** Germany
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 6.2; Win64; x64; rv:74.0) Gecko/20100101 Firefox/74.0
**Extension:** Nano Defender for Firefox 15.0.0.194

### Additional Message

The issue has still not been solved anyway and I think it deserves a closer look. It doesn't occur on the overview webpage of course, but only if you really try to download the program. Then, there is a message "Sie haben einen Adblocker aktiviert" ("You have an active adblocker"). This is the actual issue.This message is just an annoyance that should be blocked and removed. Anyway, it can luckily be bypassed quite easily, just by clicking on "Nein, weiter zum Download" ("No, go to the download") - then, you'll get to the download directly without watching the ad.

**Comment by jspenguin2017** [comment]
For https://github.com/NanoMeow/QuickReports/issues/368#issuecomment-599147780 : 
The download is fake, it downloads a third party installer instead of the official installer. The download is blocked by the uBlock badware filter, which is intended.

**Comment by jspenguin2017** [comment]
For https://github.com/NanoMeow/QuickReports/issues/368#issuecomment-599147780 : 
The download is fake, it downloads a third party installer instead of the official installer. The download is blocked by the uBlock badware filter.

**Comment by NanoMeow** [comment]
### Basic Information

**Test link:** `https://www.chip.de/downloads/VLC-player-64-Bit_53513913.html`
**Category:** anti-adblock
**Reported from:** Germany
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 6.2; Win64; x64; rv:74.0) Gecko/20100101 Firefox/74.0
**Extension:** Nano Defender for Firefox 15.0.0.194

### Additional Message

You are fully right, of course. I dislike chip.de as well, for the same reasons. But if you click on the small text "Manuelle Installation" (on the right of the big download button), then you will get a normal download without any strange chip-installers, so it would be kind if you solved the issue anyway.
By the way, please have a look at the spiegel.de issue(s) as well, there are video ads which are not (yet) blocked.

**Comment by jspenguin2017** [comment]
For https://github.com/NanoMeow/QuickReports/issues/368#issuecomment-600111084 : 
Can't reproduce. The website works correctly with the German regional filters enabled. 
There is an ads timer, should be boostable, but that's low priority.
